frontend/benefit/applicant/src/components/applications/forms/application/step2/useApplicationFormStep2.ts

```diff
@@ -35,9 +42,62 @@ type ExtendedComponentProps = {
   getDefaultSelectValue: (fieldName: keyof Application) => OptionType;
 };
 
+const ssnValidationTable = [
```

Comment:
https://www.npmjs.com/package/finnish-ssn
Could we use it?

Reply:
I replaced our implementation with this dependency.